view.go

```diff
@@ -270,6 +274,36 @@ func (v *View) newFragment(path string, slice uint64) *Fragment {
 	return frag
 }
 
+// DeleteFragment removes the fragment from the view.
+func (v *View) DeleteFragment(slice uint64) error {
+
+	fragment := v.fragments[slice]
+	if fragment == nil {
+		return ErrFragmentNotFound
+	}
+
+	v.logger().Printf("delete fragment: %d", slice)
```

Comment:
If you have access to the frame and view labels that would be informative for the log